_episodes/01-rebase.md

```diff
@@ -47,9 +50,10 @@ $ git rebase master
 ### Advantages and disadvantages
 
 - `git rebase` makes "merges" producing a linear history.
+- `git merge` resolves all conflicts in a single commit, with `git rebase` each commit being rebased needs 
+  conflict resolution.
 - `git rebase` may invalidate tests.
-- When somebody asks you to rebase your work to the work of somebody else you
-  know what this means (GitHub can do this automatically for you).
+- `git merge` creates history as it actually happened, `git rebase` shows a simpler linear history.
```

Comment:
maybe:
- git merge preserves chronology of commits and creates explicit merge commits (unless fast-forward).
- git rebase can change chronology of commits

Reply:
i agree, fixed this as well. thanks for review